external/mdal/frmts/mdal_binary_dat.cpp

```diff
@@ -129,6 +129,12 @@ void MDAL::LoaderBinaryDat::load( MDAL::Mesh *mesh, MDAL_Status *status )
   group->uri = mDatFile;
   group->isOnVertices = true;
 
+  // in TUFLOW results there could be also a special timestep (99999) with maximums
+  // we will put it into a separate dataset
+  std::shared_ptr<DatasetGroup> groupMax( new DatasetGroup() );
```

Comment:
this should be std::make_shared - but I'll fix in mdal

Reply:
thanks! is it worth to release new new MDAL and update here? I am confident that I need to update MDAL before qgis 3.4 release later when I add support to more formats.